src/index.html

```diff
@@ -5,10 +5,63 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="moyo_logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="moyo_logo"
+          >
+        </a>
+      </div>
```

Comment:
1) this div wrapper is redundant. Implement styles for a link
2) use BEM recommendations for naming
```suggestion
        <a href="#home" class="header__logo">
          <img
            src="images/logo.png"
            alt="moyo_logo"
          >
        </a>
```